src/main/scala/za/co/absa/hyperdrive/trigger/api/rest/utils/PathResolverUtil.scala

```diff
@@ -0,0 +1,27 @@
+
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.hyperdrive.trigger.api.rest.utils
+
+object PathResolverUtil {
+  def resolve(basePath: String, childPath: String): String = {
+    basePath match {
+      case bp: String if bp.isEmpty => childPath
+      case bp: String if childPath.startsWith("/") => s"${bp}${childPath.replaceFirst("/", "")}"
```

Comment:
Does this make sense? If `basePath` is `/tmp` and ` childPath` is `/some/path` then the result would be `/tmpsome/path`